Sources/PointFree/Episode/Show.swift

```diff
@@ -487,10 +487,36 @@ private let episodeInfoView = View<(EpisodePermission, Episode)> { permission, e
   div(
     [`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]]), Class.pf.colors.bg.white])],
     topLevelEpisodeInfoView.view(ep)
-    + sectionsMenu(episode: ep, permission: permission)
+      + previousEpisodes(of: ep)
+      + sectionsMenu(episode: ep, permission: permission)
   )
 }
 
+private func previousEpisodes(of ep: Episode) -> [Node] {
+  let previousEps = ep.previousEpisodes
+  guard !previousEps.isEmpty else { return [] }
+
+  return [
+    div(
+      [`class`([Class.padding([.mobile: [.top: 2], .desktop: [.top: 3]]), Class.pf.colors.bg.white])],
+      [
+        div([
+          "This episode builds on concepts introduced previously:"]),
```

Comment:
I think it would be nice if the text in this callout was the same style as the episode blurb, which I think can be done with `Class.pf.type.body.regular`.